tools/migration/cfg/migrator_1_8_0/harbor.yml.jinja

```diff
@@ -0,0 +1,113 @@
+## Configuration file of Harbor
+
+#The IP address or hostname to access admin UI and registry service.
+#DO NOT use localhost or 127.0.0.1, because Harbor needs to be accessed by external clients.
+hostname: {{hostname}}
+
+# http related comfig
+http:
+  # port for http, default is 80. If https enabled, this port will redirect to https port
+  port: 80
```

Comment:
What if the user is not configured using `80/443` port in previous setup?

Reply:
We shouldn't handle this. Because from our point of view, User must use 80 and 443 in the previous version